addon/debounce-task.js

```diff
@@ -123,7 +123,7 @@ export function debounceTask(obj, name, ...debounceArgs) {
 export function cancelDebounce(obj, name) {
   let pendingDebounces = registeredDebounces.get(obj);
 
-  if (pendingDebounces === undefined) {
+  if (!pendingDebounces || !pendingDebounces[name]) {
```

Comment:
The change seems good to me! Would you mind refactoring to something like:

```js
if (pendingDebounces === undefined || pendingDebounces[name] === undefined) {
```

I'd like to (in general) only guard against the specific values we expect (the only valid values are `undefined` and the actual expected object)...

Reply:
👍Done. I was curious about that; thanks for explaining the rationale behind checking specifically for `undefined`.